### ISSUE TITLE
Fan support

### ISSUE THREAD

**Issue by rithask** [lucavb/homebridge-esphome-ts#150]
Can you add fan support so that it'll show as a fan in Apple Home app. 
Thanks.

**Comment by lucavb** [comment]
Okay so good news, I don't see any reason why we should not be able to make this happen. I just need to figure out how the fans are behaving in esphome. And maybe I will need you as a tester. I'll update you when I am ready.

**Comment by rithask** [comment]
Sure, that's great.

**Comment by lucavb** [comment]
No, with the debug option on, the plugin is going to generate a file under `/tmp`. I would like to see that one. Although the hint of 30s is an indication. There is a timeout that matches that duration.
